app/config/usage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
+            '24h' => [
+                'period' => '1h',
+                'limit' => 24,
+                'factor' => 3600,
```

Comment:
What is the factor used for ?

Reply:
To calculate the leaps on the API endpoints (for instance the storage usage endpoint)

<img width="688" alt="image" src="https://user-images.githubusercontent.com/23742426/216277441-a40d90af-4546-432f-a767-76727cfc95be.png">